c/cert/src/qlpack.yml

```
@@ -1,4 +1,6 @@
 name: cert-c-coding-standards
```

I think this needs to be
```suggestion
name: codeql/cert-c-coding-standards
```
to fix the CI failures.